test/generators/shop_model_generator_test.rb

```diff
@@ -29,6 +33,13 @@ class ShopModelGeneratorTest < Rails::Generators::TestCase
     end
   end
 
+  test "injects into application controller" do
+    run_generator
+    assert_file "app/controllers/application_controller.rb" do |controller|
+      assert_match "def shop", controller
+    end
```

Comment:
What's in your `session[:shopify]`? 

This is the contents of the shop method:

```
@shop ||= Shop.find_by(id: session[:shopify])
```

Can you run that and let me know what happens? Its possible this method needs to be updated